src/main.js

```diff
@@ -284,21 +285,23 @@ function handleClose(streams, children, childrenInfo) {
         if (aliveChildren.length === 0) {
             exit(exitCodes);
         }
-    });
 
-    if (config.killOthers) {
```

Comment:
I see, in that case it sounds like we need a new feature that will allow us to say we should kill processes if one of them exits with a non-zero exit code. If you're interested in a future like that then I'm happy to make a port request for that. What would you like the flag to be called?